cfgov/legacy/views.py

```diff
@@ -13,9 +21,6 @@ def get_render_url(self):
     def get(self, request):
 
         self.request = request
-        if settings.DEBUG and PDFReactor is None:
-            return HttpResponse("PDF Reactor is not configured, can not render %s" % self.get_render_url())
```

Comment:
why remove this? (though the capitalization was incorrect, it should be PDFreactor)